scripts/github-actions/generate.sh

```diff
@@ -96,7 +96,14 @@ for tag in $tags; do
 						| join(" ")
 					),
 					history: ("docker history " + (.tags[0] | @sh)),
-					test: ("~/oi/test/run.sh " + (.tags[0] | @sh)),
+					test: (
+						[
+							"set -- " + (.tags[0] | @sh),
+							# https://github.com/docker-library/bashbrew/issues/46#issuecomment-1152567694 (allow local test config / tests)
+							"if [ -s .test/config.sh ]; then set -- --config ~/oi/test/config.sh --config .test/config.sh \"$@\"; fi",
```

Comment:
```suggestion
							"if [ -f ./.test/config.sh ]; then set -- --config ~/oi/test/config.sh --config ./.test/config.sh \"$@\"; fi",
```

Reply:
What's the use case for an _empty_ configuration file? :grimacing:

(I can definitely go with `./` prefix making the intent here more obvious, but before I apply that without the `-s` to `-f` change I'd like to understand what you're hoping that will do :sweat_smile:)

Reply:
Yeah I agree I had to investigate about `-s`
Checking file is better

Reply:
Perfect, thanks for confirming!  Updated to add `./` for clarity :+1: